test/integration/etcd/data.go

```diff
@@ -231,38 +231,6 @@ func GetEtcdStorageDataForNamespace(namespace string) map[schema.GroupVersionRes
 		},
 		// --
 
-		// k8s.io/kubernetes/pkg/apis/flowcontrol/v1beta1
```

Comment:
note to self that this needs scrutiny for why it's being removed as part of these changes.

Reply:
these are apis removed before the binary version used in the test.

Reply:
to be restored.  this is necessary to ensure that prior versions will be interoperable with emulated versions of the current level.